SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -278,6 +282,11 @@ void USpatialActorChannel::DeleteEntityIfAuthoritative()
 			UE_LOG(LogSpatialActorChannel, Error, TEXT("DeleteEntityIfAuthoritative called on actor channel with null actor - entity id (%lld)"), EntityId);
 		}
 	}
+	else if (bCreatedEntity) // We have not gained authority yet
+	{
+		Actor->SetReplicates(false);
```

Comment:
There's a check above for actor !- nullptr, could we do the actor != nullptr check first, then do the has auth check?